src/main/resources/modernizer.xml

```diff
@@ -918,4 +918,126 @@ violation names use the same format that javap emits.
     <comment>Prefer java.time.format.DateTimeFormatter</comment>
   </violation>
 
+  <violation>
+    <name>java/io/PrintStream."&lt;init&gt;":(Ljava/io/File;Ljava/lang/String;)V</name>
+    <version>1.10</version>
+    <comment>Prefer java.io.PrintStream(File, Charset)</comment>
+  </violation>
+
+  <violation>
+    <name>java/io/PrintStream."&lt;init&gt;":(Ljava/lang/String;Ljava/lang/String;)V</name>
+    <version>1.10</version>
+    <comment>Prefer java.io.PrintStream(String, Charset)</comment>
+  </violation>
+
+  <violation>
+    <name>java/io/PrintWriter."&lt;init&gt;":(Ljava/io/File;Ljava/lang/String;)V</name>
+    <version>1.10</version>
+    <comment>Prefer java.io.PrintWriter(File, Charset)</comment>
+  </violation>
+
+  <violation>
+    <name>java/io/PrintWriter."&lt;init&gt;":(Ljava/lang/String;Ljava/lang/String;)V</name>
+    <version>1.10</version>
+    <comment>Prefer java.io.PrintWriter(String, Charset)</comment>
+  </violation>
+
+  <violation>
+    <name>java/io/ByteArrayOutputStream.toString:(Ljava/lang/String;)Ljava/lang/String;</name>
+    <version>1.10</version>
+    <comment>Prefer java.io.ByteArrayOutputStream.toString(Charset)</comment>
+  </violation>
+
+
+  <violation>
+    <name>java/net/URLDecoder.decode:(Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;</name>
+    <version>1.10</version>
+    <comment>Prefer java.net.URLDecoder.decode(String, Charset)</comment>
+  </violation>
+
+  <violation>
+    <name>java/net/URLEncoder.encode:(Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;</name>
+    <version>1.10</version>
+    <comment>Prefer java.net.URLEncoder.encode(String, Charset)</comment>
+  </violation>
+
+
```

Comment:
Here and above: are the extra empty lines on purpose? :)

(Asking because I didn't see a clear pattern.)

Reply:
Removed.